tests/genomics_tests/test_genomic_cvl_pipeline.py

```diff
@@ -740,22 +782,33 @@ def test_w1il_manifest_generation(self, cloud_task_mock, sql_exporter_class_mock
                 self.expected_w1il_row(self.ny_set_member, self.ny_validation_metrics, self.ny_summary),
                 self.expected_w1il_row(self.male_set_member, self.male_validation_metrics, self.male_summary),
                 self.expected_w1il_row(self.hdr_and_pgx_set_member, self.hdr_and_pgx_validation_metrics,
-                                       self.hdr_and_pgx_summary)
+                                       self.hdr_and_pgx_summary),
+                self.expected_w1il_row(self.bcm_pipeline_set_member, self.bcm_pipeline_validation_metrics,
+                                       self.bcm_pipeline_summary)
             ]
         )
         self.assert_manifest_has_rows(
             manifest_cloud_writer_mock=co_w1il_manifest,
             expected_rows=[
-                self.expected_w1il_row(self.co_set_member, self.co_validation_metrics, self.co_summary)
+                self.expected_w1il_row(self.co_set_member, self.co_validation_metrics, self.co_summary),
+                self.expected_w1il_row(self.co_pipeline_set_member, self.co_pipeline_validation_metrics,
+                                       self.co_pipeline_summary)
+            ]
+        )
+        self.assert_manifest_has_rows(
+            manifest_cloud_writer_mock=uw_w1il_manifest,
+            expected_rows=[
+                self.expected_w1il_row(self.uw_pipeline_set_member, self.uw_pipeline_validation_metrics_3_4_12,
+                                       self.uw_pipeline_summary)
```

Comment:
Can you modify this test and the expected method test to make sure your replace function in the query is appending the `archive` string?

Reply:
I updated the expected rows & generated participants to do what I think you're asking. Let me know if that's not what you were looking for.